ansible/files/bin/snikket-turn-addresses

```diff
@@ -18,19 +20,11 @@ if not ip_addr.private then
 	os.exit(0)
 end
 
--- follow at most 3 CNAMEs (arbitrary choice)
-local final_record = arg[1];
-for i=1,3 do
-	local reply = dns.lookup(final_record, "CNAME");
-	if not reply or #reply== 0 then
-		break
-	end
-	local dest = reply[1].cname;
-	io.stderr:write(string.format("following CNAME %s -> %s\n", final_record, dest))
-	final_record = dest;
-end
-local dns_record = dns.lookup(final_record, ip_addr.proto == "IPv6" and "AAAA" or "A")
+local dns_record = dns.lookup(host_name, ip_addr.proto == "IPv6" and "AAAA" or "A")
 if not dns_record or #dns_record == 0 then
+	io.stderr:write(("ERROR: No external address found for %s on %s\n"):format(ip_addr.proto, host_name));
```

Comment:
There are more possible error conditions here that might warrant coverage. E.g. `not dns_record` would generally indicate some sort of syntax error with the `host_name` or possibly some configuration issue with libunbound.